src/jwplayer/players/shared/JwPlayerWrapper.tsx

```diff
@@ -178,11 +154,7 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({
 		}
 	};
 
-	return (
-		<div className={className}>
-			<div id={jwPlayerContainerEmbedId} />
-		</div>
-	);
+	return <div className={className}>{jwpWrapperInitialized && <div id={jwPlayerContainerEmbedId} />}</div>;
```

Comment:
why did you add `jwpWrapperInitialized` check here?

Reply:
I thought we didn't want to render the `<div />` before we have signal from the hook JWP has been initialised. But since the flag only means that the `initPlayer()` was called (it doesn't mean the JWP script has been loaded or the sponsored videos has been retrieved) I should change it and just execute the `useBeforeJwpWrapperRendered()` hook 🤔 Thanks for catching! :)